agent/agent-gc-monitor/gc-monitor-tests/src/test/java/com/microsoft/gcmonitortests/VariousCollectorsTest.java

```diff
@@ -20,6 +20,7 @@
 import org.junit.jupiter.api.condition.EnabledForJreRange;
 import org.junit.jupiter.api.condition.EnabledOnOs;
 
+@Disabled // TODO (trask) too flaky since we stopped re-running tests automatically on failure
```

Comment:
I ran out of time to look into the flaky failures here

I think we should just disable it for now so that it doesn't impact the release, I've created a work item to track it